### PR TITLE
Fix the extra copy warning and float abs warning.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

**Review by jievince — APPROVED** [review]

**Review by czpmango — APPROVED** [review]

File: src/validator/LookupValidator.cpp [hunk 1]
```diff
@@ -416,7 +416,7 @@ StatusOr<Value> LookupValidator::checkConstExpr(Expression* expr,
         if ((leftIsAE && (kind == Expression::Kind::kRelGE || kind == Expression::Kind::kRelLT)) ||
             (!leftIsAE && (kind == Expression::Kind::kRelGT || kind == Expression::Kind::kRelLE))) {
             // edge case col1 >= 40.0, no need to round up
-            if (abs(f - iCeil) < kEpsilon) {
+            if (std::abs(f - iCeil) < kEpsilon) {
```
Comment (Aiee) [inline src/validator/LookupValidator.cpp:419]: Should use `fabs()`

Comment (Shylock-Hg) [reply src/validator/LookupValidator.cpp:419]: It's ok too.

Comment (Shylock-Hg) [reply src/validator/LookupValidator.cpp:419]: See <iframe width="800px" height="200px" src="https://godbolt.org/e#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAM1QDsCBlZAQwBtMQBGAFlJvoCqAZ0wAFAB4gA5AAYppAFZdSrZrVDIApACYAQjt2kR7ZATx1KmWugDCqVgFcAtrS4AOUlfQAZPLUwAcs4ARpjEXDKkAA6oQoTmtHaOLu7RsfF0vv5BTqHhnJHGmKYJDATMxARJzq6cHkUldGUVBFmBIWERRuWV1Sl13S1tOXkRAJRGqA7EyBxSOgDMfsiOWADUmgs2AG7FRMSb2JoyAIKLy6uYG1vmQgTEmMxOh8dn2ku0Kw7rmzaseE5CEIXqdzp9LtcbCwhCJKiCzqc/AQ1k5mH4IGMNgB2fSg05rAlrKisVDMZF4a4AETWAFptAA6BabXEnQlrO7oEAoKbI36/dkETkgZjBIQQPCYvlbAVCrysZmvNkcrloBy8rb8kViiWQ/nKkByhV41mEh4Eaa0NYyI1nLGUqQTVjSACs8lcsnkqGkNgMBnZUxmV0WnHkBGkcjGEwA1iAFgtGXHE0mkwA2FTSbhu8Oe6TyIQgSJhj0TOCwJBoJxRPDsMgUCAVqs1lCqdQFGSRKjVghhfMQYLZ0jBPwVACe0hDpArTisBAA8rRWGOPaQsKj1OwB/gHiVdvnl5hxMU1XM5OR6Jgncv/sFiKO7Fhx6HiADHxN%2BIwWBueHw6ARhGJJGXJROBUNQNF9PQVDwUV4AmVAojMOg9xpDlNkpLQ9AMTgsTzC89gSSxrD6WpPGsYYOnyNI4kQxJ7BqZQYmohJyNyToQIaGjml6Oj%2BiMPDGloLjWj8dpWMou4WmI5QJMqFjRk4CYhADWYuEdF0s2XL0pHENwUxpFNuDWFYwLWAp6Rkcy1ggXBCBIDZ3hAtY7EraswnshYFKciDdFDbNI1IGM4wTZMQoWNNL0zUh3VPLS8wLUgiwjUhS0QEAeSiNVyEoBtXMozB8H2ZR3yYNgOG/d9/wkAcAHdbyiV90ykV0ooHLTgzWarCAQNYdL0gyjJbYBTIskbfOLaNY3jULQsayLopzKQ4sLPy1KkbQNJi3MEpW0hdmIOILG4IA"></iframe>

Comment (Shylock-Hg) [reply src/validator/LookupValidator.cpp:419]: See https://godbolt.org/z/fG3xcnGcq

**Pull request merged**